src/style.css

```
@@ -1 +1,62 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 40px 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 0;
+}
+
+.menu {
+  display: flex;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.menuItem {
```

don't use camelCase for classnames
```suggestion
.menu-item {
```